src/Ticket.php

```diff
@@ -4321,7 +4321,8 @@ public function showForm($ID, array $options = [])
             $options['_skip_promoted_fields'] = false;
         }
 
-        if (!$ID) {
+        if (!$ID && !array_key_exists('_glpi_csrf_token', $options['_saved'])) {
```

Comment:
Checking the `_glpi_csrf_token` token may not work as expected in some edge cases. Indeed, we could imagine a feature that would introduce pre-selection steps that result in calling the `Ticket::showForm()` method with some predefined options based on a submitted form.

Code would be most resilient with the introduction of a `CommonDBTM::hasSavedInput()` method, that would return the result of `isset($_SESSION['saveInput'][$this->getType()])`.